src/smc-util/compute-images.coffee

```diff
@@ -0,0 +1,19 @@
+{defaults, required} = require('smc-util/misc')
```

Comment:
I just moved this to smc-util, so COMPUTE_IMAGES is already compiled into the client.  This is better since:

- there isn't an async race condition when querying for the images to put the menu entries in project settings,
- less load on the hub servers.
- it's far *easier* to update the client frontends than the hubs, which we basically rarely want to restart, since that destabilizes things for all users, and increased db load a lot. 
- I bet the list of compute image tags won't change that often, but if it does, we'll have a better solution.

Reply:
well, I wanted these entries to be either in the database (that's what I meant with that they aren't there yet) and there might be new ones. The list here is just very generic, and e.g. "old" makes no sense right now. What I wasn't sure is if it is a good idea to put it into the database at all. A much more direct way would be to query the image server.